components/testing/COMPONENT_FPGA_CI_TEST_SHIELD/DynamicPinList.cpp

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2019, Arm Limited and affiliates.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "DynamicPinList.h"
+
+DynamicPinList::DynamicPinList()
+{
+
+}
+
+DynamicPinList::DynamicPinList(const PinList *pin_list)
```

Comment:
I believe some changes here are already on master - bad rebase? Please review.

This file should not be part of this PR.

Reply:
You are right. The "components/testing/" folder should not be changed. I am not sure where the changes come from...
Do you have any suggestion for me or could you continue with the other changes from this PR?

Reply:
Not certain what you did. You can cleanup few ways.

You can remove files not changed by you and amend commit. As this is a new target, only few files needed, might be faster even to start from master and add new files and changes t oyour target.

Reply:
I tried to remove the unchanged files from my branch. Could you please check if this is ok now?
Thanks for your support.

Reply:
Unfortunately not, still shown here as changed - file removal not related